assets/scripts/script.js

```diff
@@ -1 +1,32 @@
-/* jshint esversion: 11 */
\ No newline at end of file
+// Save settings to localstorage
+document.getElementById('settings-form').addEventListener('submit', function(event) {
```

Comment:
looks good, looks specific to settings page , can we move to a new settings.js, then link that in settings.html - this will help avoid any side effects since script.js is linnked on ev ery page